docs/guides/initial-query-data.md

```diff
@@ -49,44 +65,35 @@ By default, `initialData` is treated as totally fresh, as if it were just fetche
 
   ```js
   // Show initialTodos immeidately, but won't refetch until another interaction event is encountered after 1000 ms
-  const result = useQuery('todos', () => fetch('/todos'), {
-    initialData: initialTodos,
-    staleTime: 60 * 1000 // 1 minute
-    // This could be 10 seconds ago or 10 minutes ago
-    initialDataUpdatedAt: initialTodosUpdatedTimestamp // eg. 1608412420052
-  })
+  function useTodosQuery() {
+    const initialTodos = [...];
+
+    return useQuery("todos", () => fetch("/todos"), {
+      initialData: initialTodos,
+      staleTime: 60 * 1000 // 1 minute
+      // This could be 10 seconds ago or 10 minutes ago
+      initialDataUpdatedAt: initialTodosUpdatedTimestamp // eg. 1608412420052
+    });
+  }
   ```
 
   This option allows the staleTime to be used for its original purpose, determining how fresh the data needs to be, while also allowing the data to be refetched on mount if the `initialData` is older than the `staleTime`.  
   In the example above, our data needs to be fresh within 1 minute, and we can hint to the query when the `initialData` was last updated so the query can decide for itself whether the data needs to be refetched again or not.
 
 ?> If you would rather treat your data as `prefetched data`, we recommend that you use the `prefetchQuery` or `fetchQuery` APIs to populate the cache beforehand, thus letting you configure your `staleTime` independently from your `initialData`
 
-### Initial Data Function
-
-If the process for accessing a query's initial data is intensive or just not something you want to perform on every render, you can pass a function as the `initialData` value.  
-This function will be executed only once when the query is initialized, saving you precious memory and/or CPU:
-
-```js
-const result = useQuery("todos", () => fetch("/todos"), {
-  initialData: () => {
-    return getExpensiveTodos();
-  },
-});
-```
-
```

Comment:
It doesn't make sense in Vue to provide initial data as a function because composables in Vue are executed only once (per component instance).

Hence, I think, it doesn't worth it mentioning such functionality